src/components/ContentHighlights/HighlightStepper/ContentSearchResultCard.jsx

```diff
@@ -11,13 +11,16 @@ const ContentSearchResultCard = ({ original }) => {
     cardImageUrl,
     originalImageUrl,
   } = original;
-
+  const extras = {
+    firstEnrollablePaidSeatPrice: original.firstEnrollablePaidSeatPrice,
+  };
```

Comment:
[curious] Any reason why we couldn't pass a `price` prop to `ContentHighlightCardItem` vs. passing this `extras` prop as an object?

```jsx
<ContentHighlightCardItem
  title={title}
  contentType={contentType}
  partners={partners}
  cardImageUrl={cardImageUrl || originalImageUrl}
  price={firstEnrollablePaidSeatPrice}
/>
```